src/layout-rect.js

```diff
@@ -229,3 +229,19 @@ export function layoutRectEquals(r1, r2) {
   return r1.left == r2.left && r1.top == r2.top &&
       r1.width == r2.width && r1.height == r2.height;
 }
+
+/**
+ * @param {LayoutMarginsChangeDef} marginsChange
+ * @return {LayoutMarginsChangeDef}
+ */
+export function cloneLayoutMarginsChangeDef(marginsChange) {
+  if (marginsChange === undefined) {
```

Comment:
Given the type annotation on the param, marginsChange can be null. So:
if (!marginsChange) {
  return marginsChange;
}
Would handle both undefined and null.